training/vision/components/instance_segmentation/spec.yaml

```diff
@@ -15,51 +15,187 @@ inputs:
     description: Validation data
     type: mltable
     optional: true
+  ams_gradient:
+    description: Enable ams_gradient when optimizer is adam or adamw.
+    type: boolean
+    optional: true
+  beta1:
+    description: Value of beta1 when optimizer is adam or adamw. Must be a float in the range [0, 1].
+    type: number
+    optional: true
+    min: 0
+    max: 1
+  beta2:
+    description: Value of beta2 when optimizer is adam or adamw. Must be a float in the range [0, 1].
+    type: number
+    optional: true
+    min: 0
+    max: 1
+  box_detections_per_image:
+    description: Maximum number of detections per image, for all classes. Must be a positive integer.
+    type: integer
+    optional: true
+    min: 1
+  box_score_threshold:
+    description: During inference, only return proposals with a classification score greater than box_score_threshold. Must be a float in the range [0, 1].
+    type: number
+    optional: true
+    min: 0
+    max: 1
+  checkpoint_frequency:
+    description: Frequency to store model checkpoints. Must be a positive integer.
+    type: integer
+    optional: true
+    min: 0
+  checkpoint_run_id:
+    description: The run ID of the experiment that has a pretrained checkpoint for incremental training.
+    type: string
+    optional: true
+  early_stopping:
+    description: Enable early stopping logic during training.
+    type: boolean
+    optional: true
+  early_stopping_patience:
+    description: Minimum number of epochs or validation evaluations with no primary metric improvement before the run is stopped. Must be a positive integer.
+    type: integer
+    optional: true
+    min: 1
+  early_stopping_delay:
+    description: Minimum number of epochs or validation evaluations to wait before primary metric improvement is tracked for early stopping. Must be a positive integer.
+    type: integer
+    optional: true
+    min: 1
+  evaluation_frequency:
+    description: Frequency to evaluate validation dataset to get metric scores. Must be a positive integer.
+    type: integer
+    optional: true
+    min: 1
+  gradient_accumulation_step:
+    description: Gradient accumulation means running a configured number of gradient_accumulation_step without updating the model weights while accumulating the gradients of those steps, and then using the accumulated gradients to compute the weight updates. Must be a positive integer.
```

Comment:
Should we align things with image_classification_subgraph.yaml in AzureMlCli repo ?

Reply:
done.